model/src/test/java/com/javahelp/model/user/ClientUserInfoTest.java

```diff
@@ -51,6 +53,32 @@ public void testSetAddress() {
         assertEquals("Bahen Center", c.getAddress());
     }
 
+    @Test(timeout = 50)
+    public void testSetSingleAttributeExist() {
+        c.setAttribute("Gender", "Male");
+        assertEquals("Male", c.getSingleAttribute("Gender"));
+    }
+
+    @Test(timeout = 50)
+    public void testSetSingleAttributeNonExist() {
+        c.setAttribute("Gender", "Male");
+        assertEquals("-1", c.getSingleAttribute("Age"));
+    }
+
+    @Test(timeout = 50)
+    public void testSetMultipleAttribute() {
+        c.setAttribute("Gender", "Male");
```

Comment:
If `Gender` enum is approved, then this line should be Gender.MALE, am I right? Just in case when this is merged before `Gender` PR.